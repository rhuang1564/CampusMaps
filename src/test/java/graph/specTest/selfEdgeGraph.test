# Tests adding a single node to the graph with a self edge

# Create a graph
CreateGraph A

# Add a node
AddNode A node1

# Add an edge
AddEdge A node1 node1 edge1

# Print the nodes in the graph and checks its edges
ListNodes A
ListChildren A node1