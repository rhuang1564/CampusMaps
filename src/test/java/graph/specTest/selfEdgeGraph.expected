# Tests adding a single node to the graph with a self edge

# Create a graph
created graph A

# Add a node
added node node1 to A

# Add an edge
added edge edge1 from node1 to node1 in A

# Print the nodes in the graph and checks its edges
A contains: node1
the children of node1 in A are: node1(edge1)